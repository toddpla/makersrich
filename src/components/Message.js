import React, { Component } from 'react';
import signs from '../data/maps/level1/signs'

class Message extends Component {
  render() {
    return (
      <div>
        {this.props.message}
      </div>
    )
  }
}
export default Message
