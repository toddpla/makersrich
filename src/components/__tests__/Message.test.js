import React from 'react'
import Message from '../Message'
import renderer from 'react-test-renderer';
import ShallowRenderer from 'react-test-renderer/shallow';


test('renders correctly', () => {
  const wrapper = renderer.create(<Message message={'message'} />)
  expect(wrapper).toMatchSnapshot()
});

test('renders with a message prop', () => {

  const renderer = new ShallowRenderer()
  renderer.render(<Message message={'message'} />)
  const result = renderer.getRenderOutput()
  console.log(result.props);
  expect(result.props.children).toEqual("message")

})
