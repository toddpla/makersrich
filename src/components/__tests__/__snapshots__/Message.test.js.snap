// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="message"
>
  message
  <div
    className="message-item undefined"
  />
</div>
`;
